lib/reek/cli/command/report_command.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'commandable'
+require_relative '../../examiner'
+
+module Reek
+  module CLI
+    module Command
+      #
+      # A command to collect smells from a set of sources and write them out in
+      # text report format.
+      #
+      class ReportCommand
+        include Commandable
+
+        def execute(app)
+          scan_for_smells app
+          reporter.smells? ? app.report_smells : app.report_success
```

Comment:
I'm not really happy with the [object oriented clusterfuck pattern](https://github.com/troessner/reek/issues/775) used here but this would be something for another PR since this is the way it is right now.